packages/itwinui-css/src/breadcrumbs/breadcrumbs.scss

```diff
@@ -44,6 +44,7 @@
 
   > :any-link {
     @include iui-anchor;
+    box-sizing: border-box;
```

Comment:
I don't understand why this was added instead of redefining the width of the link to fix the visual issues.

Reply:
In this case, only the "generic anchor" breadcrumbs example was failing because it did not have a class name that begins with `"iui-"`. The method I'm using to set `box-sizing: border-box` only defines it on elements with class names starting with `"iui-"`. `"iui-anchor"` gets `border-box`, but `:any-link` doesn't, so I'm manually including it. Also, Vykintas said that the "generic anchor" example was going to be deprecated soonish and that this solution was fine for that little time in between.
![image](https://user-images.githubusercontent.com/37936169/172942011-b7d4e0d5-8fde-4679-b7e4-81ea38edaec5.png)